egs/fisher_callhome_spanish/st1/conf/tuning/train_pytorch_conformer_kernel15_bpe_ctc_asr0.3.yaml

```diff
@@ -53,6 +53,7 @@ report-wer: true
 transformer-encoder-pos-enc-layer-type: rel_pos
 transformer-encoder-selfattn-layer-type: rel_selfattn
 transformer-encoder-activation-type: swish
+rel-pos-type: latest
```

Comment:
This is not necessary because `latest` is used by default now.

Reply:
Hi, @hirofumi0810,
Thanks so much for the review. To be compatible with the pre-trained model, the default setting is `legacy RPE`. Because when reading previous models, even `args.rel_pos_type=None`, the function `fill_missing_args()` will assign a default value to it. If we use `latest` as default, people have to manually write `args.rel_pos_type=legacy` in `model.json`.

Reply:
I see. The current default is `legacy`.